app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
```

Comment:
In this task you use only "date" from "datetime" library, so you can use this construction: "from datetime import date". And use it like this: "date.today()"